apps/native-component-list/src/screens/Video/VideoScreen.tsx

```diff
@@ -4,6 +4,14 @@ import { optionalRequire } from '../../navigation/routeBuilder';
 import ComponentListScreen, { ListElement } from '../ComponentListScreen';
 
 export const VideoScreens = [
+  {
+    name: 'Audio Options',
+    route: 'video/audio',
+    options: {},
+    getComponent() {
+      return optionalRequire(() => require('./VideoAudioScreen'));
+    },
+  },
```

Comment:
I would recommend to have this list in the order of importance. Imho "Sources", "Video Events", "Playback Controls" should definitely go first. I guess you originally wanted the alphabetical order (by screen name), but I feel like for QA testers it makes more sense to go from the top to the bottom and check basic functionalities first. For example, if something is broken in `VideoAudioScreen`, it doesn't necessarily mean that the audio mixing is broken, it could have been some basic feature that caused this so you would rather like to know that earlier.
It's the same with tests, we usually start from basic to more complex cases.